src/renderer/InvestJob.js

```diff
@@ -43,83 +43,82 @@ export default class InvestJob {
     return InvestJob.getJobStore();
   }
 
-  /**
-   * @param {object} obj - the metadata property
-   * @param {string} obj.modelRunName - name to be passed to `invest run`
-   * @param {string} obj.modelHumanName - colloquial name of the invest model
-   * @param {object} obj.argsValues - an invest "args dict" with initial values
-   * @param {string} obj.logfile - path to an existing invest logfile
-   * @param {'running'|'success'|'error'} obj.status - status of the invest process
-   */
-  constructor(
-    {
-      modelRunName,
-      modelHumanName,
-      argsValues,
-      logfile,
-      status,
-    }
-  ) {
-    this.metadata = {};
-    this.metadata.modelRunName = modelRunName;
-    this.metadata.modelHumanName = modelHumanName;
-    this.metadata.argsValues = argsValues;
-    this.metadata.logfile = logfile;
-    this.metadata.status = status;
-    this.metadata.workspaceHash = null;
-
-    this.save = this.save.bind(this);
-    this.setProperty = this.setProperty.bind(this);
-    this.setWorkspaceHash = this.setWorkspaceHash.bind(this);
-  }
-
-  setWorkspaceHash() {
-    if (this.metadata.argsValues.workspace_dir
-        && this.metadata.modelRunName) {
-      this.metadata.workspaceHash = crypto.createHash('sha1').update(
-        `${this.metadata.modelRunName}
-         ${JSON.stringify(this.metadata.argsValues.workspace_dir)}
-         ${JSON.stringify(this.metadata.argsValues.results_suffix)}`
+  static getWorkspaceHash(modelRunName, workspaceDir, resultsSuffix) {
```

Comment:
So `getWorkspaceHash` is just a name bump from `setWorkspaceHash`. Just curious why you felt it was more a 'getter' instead of a 'setter'?

Reply:
Now it returns the value, whereas before it actually set the value.

Reply:
Ha, of course!